KutokAccounting.Services/Vendors/VendorService.cs

```diff
@@ -8,24 +8,24 @@ namespace KutokAccounting.Services.Vendors;
 public sealed class VendorService : IVendorService
 {
     private readonly ILogger<VendorService> _logger;
-    private readonly QueryParametersValidator _queryValidator;
+    private readonly VendorQueryParametersValidator _vendorQueryValidator;
```

Comment:
До-речі можна їх використовувати через інтерфейс
```suggestion
    private readonly IValidator<QueryParameters> _vendorQueryValidator;
```
Але має бути реєстрація на цей інтерфейс